docs/Git_Reference.md

```diff
@@ -0,0 +1,117 @@
+[**Table of Contents**](index.md)
+
+# Git Info
+
+![git info](md_Graphics/git_sm.jpg)
+
+## Getting Started
+* [Git Basics](https://git-scm.com/book/en/v2/Getting-Started-Git-Basics)
+* [Git Book](https://git-scm.com/book/en/v2)
+* [Git Bootcamp and Cheat Sheet](https://docs.python.org/devguide/gitbootcamp.html)
+
+### General Information
+* Snapshots, not Differences.
+
+* Nearly every operation is local.
+
+* Integrity - SHA-1 Hash (40 Characters).
+
+* 3 States that files can reside in:
+    * Committed - Safely stored in the local database.
+    * Modified - Changed, not commited.
+    * Staged - Marked to go into the next commit.
+
+* fetch imports commits from a remote repo into your local repo.
+* Commits are stored as remote branches instead of normal local branches.
+* pull = fetch + merge
+* git pull --rebase  origin   ← Same as git pull but instead of git merge, git rebase
+* git pull --rebase  origin  == fetch + rebase
+* Pull Request is a mechanism for a developer to notify team members that they have completed a feature.
+
+## Useful git Commands
+
+|Git task|	Notes|	Git commands|
+|--------|-------|--------------|
+| config | List the current config file |  git config --list |
+| | Configure the author name and email address to be used with your commits. Note that Git strips some characters (for example trailing periods) from user.name.| git config --global user.name "Sam Smith"|
+| | |git config --global user.email sam@example.com|
+| | |git config --global http.proxy http://my-proxy.com |
+| | | git config --global https.proxy https://my-proxy.com |
+| | Alias for pull to do a fetch + rebase (no merge) | pull.rebase=true |
```

Comment:
This missing git config prefix like the others.

Reply:
fixed